pkg/awsconfig/awsconfig.go

```diff
@@ -113,13 +113,13 @@ func (p *CredentialsProvider) Load() (*AWSCredentials, error) {
 }
 
 // Expired checks if the current credentials are expired
-func (p *CredentialsProvider) Expired() bool {
+func (p *CredentialsProvider) Expired() (bool, *time.Time) {
```

Comment:
The function name doesn't really do what it says now.
Why not create a new method `ExpiresAt` to return the `*time.Time` instead?